### ISSUE TITLE
Icon: fix aria-hidden

### ISSUE THREAD

**Issue by jrsowles** [jrsowles/react-building-blocks#8]
Currently, the aria-hidden attribute is always set to true. It should be set to false if the Icon component has a click handler.
